app/Http/Controllers/SecretariatController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Semester;
+
+class SecretariatController extends Controller
+{
+    public function list()
```

Comment:
Shouldn't we rename the function for better readability? eg listActiveUsers?

Reply:
I just added this as an example and used it for testing.
Later, when the concept will be actually used, we can rename/change it.